internal/server/server.go

```diff
@@ -76,14 +78,42 @@ func (s *SOMASServer) logf(format string, a ...interface{}) {
 }
 
 func initEnvironment() *disasters.Environment {
+	var cp = disasters.Commonpool{900, 1000}
 	_, clientMap := getClientInfosAndMapFromRegisteredClients(baseclient.RegisteredClients)
 	islandIDs := make([]shared.ClientID, 0, len(clientMap))
 	for id := range clientMap {
 		islandIDs = append(islandIDs, id)
 	}
 	xBounds := [2]float64{0, 10}
 	yBounds := [2]float64{0, 10}
-	dp := disasters.DisasterParameters{GlobalProb: 0.1, SpatialPDF: "uniform", MagnitudeLambda: 1.0}
-	env, _ := disasters.InitEnvironment(islandIDs, xBounds, yBounds, dp)
+	dp := disasters.DisasterParameters{GlobalProb: 1, SpatialPDF: "uniform", MagnitudeLambda: 1.0}
+	env, _ := disasters.InitEnvironment(islandIDs, xBounds, yBounds, dp, cp)
 	return env
 }
+
+//islandDistribute distributes leftover resource after disaster's mitigation
+func (s *SOMASServer) islandDistribute(leftover uint, disasterReport string) { //receives 

```

Comment:
So after a disaster, what is left in the common pool is emptied and distributed to the islands?

Reply:
If this is the case, it will cause problems for IIGO as the IIGO session happens at the beginning of a turn and we use the common pool to pay salaries. If salaries aren't paid then IIGO doesn't run. Maybe we can discuss this? i suppose it's more of a design issue :)

Reply:
I agree with @preetl  -- I was thinking common pool is not getting re-distributed at all BUT some amount from it can be allocated by doing an allocation request to the President.... we gotta discuss this soon :)